src/constants/duel_constants.asm

```diff
@@ -117,9 +117,13 @@ ASLEEP          EQU $02
 PARALYZED       EQU $03
 POISONED        EQU $80
 DOUBLE_POISONED EQU $c0
+DOUBLE_POISONED_F EQU 6
+POISONED_F      EQU 7
```

Comment:
We try to be particular about keeping whitespace clean and organized.
So I would recommend that you add a blank line above `DOUBLE_POISONED_F EQU 6` and add a few spaces after `POISONED_F` to keep the `EQU` vertically aligned with the `EQU` for `DOUBLE_POISONED_F`.